src/styles/NewSnippet.styl

```diff
@@ -96,3 +97,25 @@ lang-bar-width = 230px
         border-left: 5px solid snippet-border
         border-bottom: none
         background-color: language-bar-bg-active
+
+@media (max-width: 768px)
```

Comment:
Why `768px`? Is this some sort of "best practices" value? Can you imagine some other value here? If so, I'd suggest to make it constant and use constant here and everywhere around.